src/main/java/eli/command/Command.java

```diff
@@ -0,0 +1,41 @@
+package eli.command;
+
+import eli.exception.EliException;
+import eli.storage.Storage;
+import eli.task.Task;
+import eli.task.TaskList;
+import eli.ui.Ui;
+
+/**
+ * Abstract class representing a Command that can be executed.
+ * Concrete command classes should extend this class and implement the execute method.
+ */
+public abstract class Command {
+  protected boolean isExit;
+
+  /**
+   * Constructor for Command.
```

Comment:
Additional explanation in the commenton why you set `isExit` to false would be beneficial!